server/routes/book.routes.js

```diff
@@ -6,8 +6,7 @@ import validateBook from '../middleware/validations/bookValidator';
 
 const routes = Router();
 
-routes.post('/add', isLoggedIn, validateBook, bookController.createBook);
-// routes.delete('/delete', );
-// routes.put('/update', );
+routes.post('/', isLoggedIn, validateBook, bookController.createBook);
+routes.delete('/:bookId',isLoggedIn, bookController.deleteBook);
```

Comment:
one quick thing 

isLoggedIn middleware returns 
**user** which you can access as _res.user_

so i don't think your route or function is protected 
so you have to maybe check the type of a user 

example
```
    if (req.user.isAdmin !== true) {
        return res.status(401).json({
          status: 401,
          error: 'You are not allowed to perform this action',
        });
      }
```

Reply:
Sure thing.